### DISCUSSION TITLE
[其他]: Hugging Face Space Duplication Error

### DISCUSSION THREAD

**Discussion by kayx23** [GaiZhenbiao/ChuanhuChatGPT#548] [Q&A] [answered]
### 是否已存在现有反馈与解答？

- [X] 我确认没有已有issue或discussion，且已阅读**常见问题**。

### 错误描述

Duplicating into my own space but the process is stuck at `Building`. 

### 复现操作

Duplicating space from https://huggingface.co/spaces/JohnSmith9982/ChuanhuChatGPT

### 错误日志

Following log is observed: 

![image](https://user-images.githubusercontent.com/39619599/230027283-c649bdfa-876d-43ab-a06c-bf4dff46179f.png)

```
v859n 2023-04-05T08:24:50.795Z Error while sending telemetry: HTTPSConnectionPool(host='api.gradio.app', port=443): Max retries exceeded with url: /gradio-launched-telemetry/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0b0a48e580>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
```

Feels like this might be an error on HF side but I'm not very familiar with the platform - wondering if anyboddy can point me towards the right direction? Thanks!

**Comment by GaiZhenbiao** [comment, marked as answer]
多重启几次Space试试？尤其试试factory reboot

**Reply by kayx23** [reply]
Factory Reboot worked. Thanks!
